src/NServiceBus.AwsLambda.SQS/LambdaLog.cs

```diff
@@ -4,6 +4,8 @@ namespace NServiceBus.AwsLambda.SQS
     using Amazon.Lambda.Core;
     using NServiceBus.Logging;
 
+    using LogLevel = Logging.LogLevel;
```

Comment:
why was this necessary?

Reply:
There is a naming conflict between `Amazon.Lambda.Core.LogLevel` and `NServiceBus.Logging.LogLevel`

Reply:
was that changed with v8?